roxie/ccd/ccdserver.cpp

```diff
@@ -12132,10 +12139,7 @@ class CRoxieServerIndexWriteActivity : public CRoxieServerInternalSinkActivity,
                 }
                 reccount++;
             }
-            if(metadata)
-                builder->finish(metadata,&fileCrc);
-            else
-                builder->finish(&fileCrc);
+            builder->finish(metadata, &fileCrc);
```

Comment:
I removed the overloaded finish() method as it didn't seem to me to add any clarity or efficiency (indeed, rather the reverse).